src/lib.rs

```diff
@@ -725,6 +770,7 @@ impl Node {
 			locked_node_metrics.latest_fee_rate_cache_update_timestamp;
 		let latest_rgs_snapshot_timestamp =
 			locked_node_metrics.latest_rgs_snapshot_timestamp.map(|val| val as u64);
+		let latest_scores_sync_timestamp = locked_node_metrics.latest_scores_sync_timestamp;
```

Comment:
scoring

Reply:
My plan was to use `scoring` everywhere, but then `scoring_sync` didn't sound right? Always harder than expected, naming.

Reply:
rename to `pathfinding_scores`

Reply:
renamed to `pathfinding_scores`. It still felt that routing is too overloaded.